zapcli/cli.py

```diff
@@ -430,6 +430,54 @@ def report(zap_helper, output, output_format):
     console.info('Report saved to "{0}"'.format(output))
 
 
+@cli.command('test-alerts')
+@click.option('--config-file', '-c', default=None, type=str,
+              help='config file to use to INFO, IGNORE or FAIL warnings')
+@click.option('--config-url', '-u', default=None, type=str,
```

Comment:
My mistake, the original script support also config-url but it was a bit complicated so I didn't add it.